docs/source/gbd2019_models/risk_exposures/low_birthweight_short_gestation/index.rst

```diff
@@ -194,14 +194,65 @@ appropriate location IDs for the model you're working on):
     (sex_id=[1,2]) since we will be initializing our population using GBD's
     population data and stratifying by sex.
 
+Correcting LBWSG exposure data pulled from GBD 2019
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. important::
+
+  **The GBD 2019 exposure data for Low Birthweight and Short Gestation is wrong!**
```

Comment:
No need to be alarmist... we didn't publish the wrong value, we just stored the data in way that leads to confusion

Reply:
@aflaxman Do you want me to edit this somehow?  I just wanted to call the SWEs' attention to the fact that we need to fix the data before we use it.

Reply:
Also, how do you know we didn't publish the wrong value?

Reply:
I don't know for sure... but I'm sensitive to how this might appear if taken out of context.  I'll propose an edit
```suggestion
  **The GBD 2019 exposure data for Low Birthweight and Short Gestation is potentially misleading as currently stored!**
```

Reply:
Ok, I'll reword this. I'm also adding a Todo to add more details about the problem, to address Ali's question above.